### PR TITLE
feat(next-international): plurals with count 0 works with `#zero`

### PR DESCRIPTION
**Pull request by lindesvard**
Closes https://github.com/QuiiBz/next-international/issues/199

### CODE REVIEW

File: packages/next-international/src/common/create-t.ts [hunk 1]
```diff
@@ -27,6 +27,11 @@ export function createT<Locale extends BaseLocale, Scope extends Scopes<Locale>
       .filter(key => key.includes('#'))
       .map(key => key.split('#')[0]),
   );
+
+  function getPluralKey(count: number) {
+    if (count === 0) return 'zero';
+    return pluralRules.select(count);
+  }
   const pluralRules = new Intl.PluralRules(context.locale);
```
Comment (QuiiBz) [inline packages/next-international/src/common/create-t.ts:35]: Could you move this `pluralRules` variable before the new `getPluralKey` function? We try to keep things declared before they are used.

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks for the PR! After thinking more about this, I agree to land this change as it makes more sense - and it's not really a breaking change either.

I would like to include this as part of the 1.1.0 release: https://github.com/QuiiBz/next-international/pull/205

File: packages/next-international/src/common/create-t.ts (hunk 1 above)
Comment (lindesvard) [reply packages/next-international/src/common/create-t.ts:35]: Sure

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Could we also update the documentation for both the App & Pages Router with `#zero` and `{ count: 0 }`? We'll be ready to merge after that

Screenshot 2023-10-04 at 18 20 39

**Pull request merged**